Dockerfile

```diff
@@ -8,6 +8,9 @@ RUN zypper --non-interactive install --no-recommends \
     update-alternatives --install /usr/bin/bundle bundle /usr/bin/bundle.ruby2.5 5 && \
     update-alternatives --install /usr/bin/bundler bundler /usr/bin/bundler.ruby2.5 5
 
+RUN zypper --non-interactive install --no-recommends \
+        vim jq curl
```

Comment:
`vim` is already installed on line 6. Do we need a separated step for installing these `jq` and `curl`? Could we include those in the first `zypper install` call?

It would unnecessarily create another layer during the build process.

Reply:
Additionally, I was following the instructions on `PACKAGE.md` on a Docker container, the process breaks when building the `tar.bz2` file because the image doesn't have `bzip2`. Could you include `bzip2` on the `zypper install` statement?

Reply:
yep will do

Reply:
done